### PR TITLE
Feature delete user project

### PR DESCRIPTION
**Pull request by neelxie**
#### What does the PR do?
Add delete functionality to projects page

image

> Update
image

Kindly do note that this PR is the basis for the update functionality

### CODE REVIEW

**Comment by Mubangizi** [comment]
What does that label mean?

**Comment by neelxie** [comment]
> What does that label mean?

@Mubangizi kindly google those 2 words, and see how missing those english lessons has paid off??

File: src/components/ProjectCard/ProjectCard.css [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    color: #000000;
+    padding-top: 10px;
+
+}
+
+.ProjectDropDown {
+    position: relative;
+    display: inline-block;
+    z-index: 0;
+    margin-left: 70%;
```
Comment (mrwagaba) [inline src/components/ProjectCard/ProjectCard.css:47]: also do away with `margin` properties

File: src/components/ProjectCard/ProjectCard.css [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    opacity: 0.8;
+    transform: scale(1.02);
+}
+
+
+/*///////////// Delete Model //////////////////*/
+
+.ProjectDeleteModel{
+    position: fixed;
+    display: hidden!important;
```
Comment (mrwagaba) [inline src/components/ProjectCard/ProjectCard.css:88]: why do you have to override that value

File: src/components/ProjectCard/ProjectCard.css [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+.DeleteProjectModelResponses button{
+    display: inline;
+}
+
+.DeleteMessageDiv{
+    margin: 1.5rem;
+    text-align: center;
+}
+.DeleteErrorDiv{
+    color: red;
```
Comment (mrwagaba) [inline src/components/ProjectCard/ProjectCard.css:134]: i suggest you use color codes for color: e.g `#f00` for red, not names

File: src/components/ProjectCard/ProjectCard.css (hunk 1 above)
Comment (neelxie) [reply src/components/ProjectCard/ProjectCard.css]: @mrwagaba this has been fixed

File: src/components/ProjectCard/ProjectCard.css (hunk 2 above)
Comment (neelxie) [reply src/components/ProjectCard/ProjectCard.css:88]: yeah

File: src/components/ProjectCard/ProjectCard.css (hunk 3 above)
Comment (neelxie) [reply src/components/ProjectCard/ProjectCard.css:130]: This has been worked on. @mrwagaba

File: src/components/ProjectCard/ProjectCard.css [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+.ProjectsCard:hover {
+    transition: 0.5s;
+    box-shadow: 0 8px 16px 0px rgba(0,0,0,0.3);
+    transform: scale(0.95);
+    border: 1px solid #000;
+}
+.ProjectImageDiv{
+    margin: 0;
+    width: 100%;
+    height: 9.375rem;
```
Comment (mrwagaba) [inline src/components/ProjectCard/ProjectCard.css:14]: wow 😄

Comment (neelxie) [reply src/components/ProjectCard/ProjectCard.css:14]: I converted the Px to rem.

**Pull request merged**